docker-storage-setup.sh

```diff
@@ -891,7 +891,7 @@ fi
 # If there is no volume group specified or no root volume group, there is
 # nothing to do in terms of dealing with disks.
 if [[ -n "$DEVS" && -n "$VG" ]]; then
-  check_block_devs ${DEVS}
+  process_block_devs ${DEVS}
 
```

Comment:
Wanted to retain $DEVS, just in case we need original names of devices somewhere.

Can do following instead.

DEVS_ABS=$(process_block_devs ${DEVS})